app/models/manageiq/providers/autosde/inventory/persister/storage_manager.rb

```diff
@@ -6,67 +6,15 @@ class ManageIQ::Providers::Autosde::Inventory::Persister::StorageManager < Manag
   attr_reader  :collections
 
   def initialize_inventory_collections
-    add_san_addresses
-    add_host_initiators
-    add_volume_mappings
-    add_wwpn_candidates
-    add_host_initiator_groups
-    add_physical_storages
-    add_physical_storage_families
-    add_storage_resources
-    add_storage_services
-    add_cloud_volumes
```

Comment:
These were added so the subclass could include or exclude collections from the targeted persister.

This isn't necessary since empty targeted collections are skipped completely